dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops

```diff
@@ -1616,3 +1613,12 @@ duplicate.samesite.attribute.for.cookie.name.useraction=Check the deployment des
 duplicate.reported.for.samesite.attribute.cookie.name=CWWWC0004W: A {0} cookie name that is configured for the {1} SameSite attribute value was previously identified as a duplicate. No SameSite attribute is added for this cookie.
 duplicate.reported.for.samesite.attribute.cookie.name.explanation=A cookie name cannot be configured for multiple SameSite attribute values. This cookie name has already been identified as a duplicate.
 duplicate.reported.for.samesite.attribute.cookie.name.useraction=Check the deployment descriptor configuration to make sure the cookie name is configured for only one of the following SameSite attribute values: 'SameSiteCookies_Lax', 'SameSiteCookies_None', or 'SameSiteCookies_Strict'. 
+
+bad.request.uri\:.{0}=CWWWC0005I: The request URI has invalid or improperly encoded characters: [{0}]
+bad.request.uri\:.{0}.explanation=The request URI has invalid path parameters, invalid dot segments, or improperly encoded characters.
+bad.request.uri\:.{0}.useraction=Verify that the request URI does not contain an invalid dot or path parameter combination or any improperly encoded characters.
+
+uri.has.fragment.character=URI has a fragment [#] character; encoded [%23] or not.
+uri.has.dot.character=URI has encoded dot [%2E] character.
+uri.has.forwarslash.character=URI has encoded forwardslash [%2F] character.
```

Comment:
change so that forward slash is two words:
uri.has.forwarslash.character=URI has encoded forward slash [%2F] character.

IBM Style guide has "backslash" as one word and "forward slash" as two.

Reply:
updated